sockeye/checkpoint_decoder.py

```diff
@@ -122,7 +122,7 @@ def __init__(self,
         for factor_idx, factor in enumerate(self.targets_sentences):
             write_to_file(factor, os.path.join(model_folder, C.DECODE_REF_NAME.format(factor=factor_idx)))
 
-        self.inputs_sentences = list(zip(*self.inputs_sentences))  # type: List[List[str]]
+        self.inputs_sentences = list(zip(*self.inputs_sentences))  # type: ignore
```

Comment:
Just out of curiosity: Why is it necessary to ignore the type?

Reply:
It's one of the very old mypy issues that we were always unable to fix.

Reply:
ok. Yeah, for the CheckpointDecoder in particular I always get confused on whether we have a list of sentences with each having a list of factors or a list of factors which each have a list of sentences (the types don't help for that though).